### ISSUE TITLE
An easier way of projecting UIX canvases onto a primitive mesh 

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#1446]
### Is your feature request related to a problem? Please describe.

Right now if we want to project a UIX canvas onto a primitive mesh with all interactions it is a lengthy process first you need a camera that's pointing at the UIX canvas then you need to have that be a selective render so the camera is only rendering the UIX then you need to grab the render texture provider slap it onto the curved plane mesh and then use the laser portal components (I think that's what they're called)

this is quite a lengthy process when it could be made more user friendly and simpler to do

### Describe the solution you'd like

An easier way of projecting UIX canvases onto a primitive mesh for example something like a UIX texture you would basically slap it on to something like a curved plane mesh and all the interactions would automatically be handled

### Describe alternatives you've considered

Doing what I said above but as I said before this could be made way easier to do

### Additional Context

A comet from @TisFoolish on #1443 inspired this issue because I like the ideal of having this process be more accessible and more user friendly

### Requesters

@epicEaston197 @TisFoolish

**Comment by FlameSoulis** [comment]
Giving this a love bump, since more and more projects I've been doing involve creating the textures in a canvas and then 'render' to a texture with honestly a method that feels very antiquated. On top of that, I cannot imagine it is very resource-friendly.

If there was a way to capture the render buffer of the canvas as a texture, that would ultimately solve this issue.

**Comment by JackTheFoxOtter** [comment]
I don't think this works, there isn't really a separate render buffer for the canvas you could copy. UIX is geometry. Like it's effectively a procedural mesh. The rendering is done by the normal rendering pipeline (please correct me if I'm wrong!). If you want to put it onto a texture, you'd use a camera to record it.
